File: Day 02/operator.py
meal_cost = float(input())
tip = int(input())
tax = int(input())

tip_total = meal_cost * (tip / float(100))
tax_total = meal_cost * (tax / float(100))
total_cost = meal_cost + tip_total + tax_total
rounded_cost = round(total_cost)

message = "The total meal cost is {} dollars.".format(str(rounded_cost))
print(message)

File: Day 02/operator.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    double mealCost;
    int tip;
    int tax;

    cin >> mealCost;
    cin >> tip;
    cin >> tax;

    double tipTotal = mealCost * (tip / 100.0);
    double taxTotal = mealCost * (tax / 100.0);

    int total = round(mealCost + tipTotal + taxTotal);

    cout << "The total meal cost is " << total << " dollars." << endl;
}
